dali/pipeline/operators/resize/random_resized_crop.cc

```diff
@@ -29,14 +29,25 @@ DALI_SCHEMA(RandomResizedCrop)
   .NumOutput(1)
   .AllowMultipleInputSets()
   .AddOptionalArg("interp_type",
-      R"code(Type of interpolation used.)code",
+      R"code(Type of interpolation used. Use `min_filter` and `mag_filter` to specify
+      different filtering for downscaling and upscaling.)code",
       DALI_INTERP_LINEAR)
+  .AddOptionalArg("mag_filter", "Filter used when scaling up",
```

Comment:
if several operators are using those resampling related arguments you could register a schema `ResamplingAttr` (just the schema, no operator) and just write here `AddParent("ResamplingAttr")`.
You can look at what I did with `CropAttr` or `RandomCropAttr`.

Reply:
OK, I'll see how it works out.